includes/bootstrap.php

```diff
@@ -44,3 +44,14 @@
 		add_action( 'rest_api_init', array( 'WP_REST_Abilities_Init', 'register_routes' ) );
 	}
 }
+
+// Load assets init class for plugin bootstrap.
+if ( ! class_exists( 'WP_Abilities_Assets_Init' ) ) {
```

Comment:
Do we have in place the build step for JS assets triggered during a new GitHub release? That would be essential for the Composer package so folks could consume it as is. 

In fact, that would also be necessary for the generated zip file with the plugin that gets triggered. @justlevine might have more insights on how it works in detail.

The good news is that there a check in place which would prevent errors if the file is not found.

Reply:
I have updated the .zip artifact building to properly include the files.

After doing a bit of thinking for the composer workarounds (thanks for the details above @justlevine), I am leaning towards documenting the "post-install-cwd" step. I think this would be the most straight forward and not a huge ask of developers. It is essentially an opt-in to the client functionality. What do you think @gziolo @Jameswlepage?

(This should be considered temporary anyway if the client work is going to land with 6.9).